pkg/boot/universalpayload/utilities_arch_amd64_tinygo.go

```diff
@@ -59,10 +59,6 @@ func getPhysicalAddressSizes() (uint8, error) {
 	return 0, ErrCPUAddressNotFound
 }
 
-// Construct trampoline code before jump to entry point of FIT image.
```

Comment:
it would be nice to leave these comments in, I think.

Reply:
Updated to keep these comments.